richie/settings.py

```diff
@@ -256,14 +257,36 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
     CMS_PERMISSION = True
 
     CMS_PLACEHOLDER_CONF = {
-        "courses/cms/course_detail.html course_video": {
+        # Course detail
+        "courses/cms/course_detail.html course_teaser": {
             "name": _("Course teaser"),
             "plugins": ["VideoPlayerPlugin"],
             "limits": {"VideoPlayerPlugin": 1},
         },
-        "courses/cms/course_detail.html course_syllabus": {  # all plugins allowed
-            "name": _("Course Syllabus")
+        "courses/cms/course_detail.html course_syllabus": {
+            "name": _("Course Syllabus"),
+            # Enforce Section plugin usage at first level
+            "plugins": ["SectionPlugin"],
         },
+        "courses/cms/course_detail.html course_format": {
+            "name": _("Format"),
+            "plugins": ["TextPlugin"],
+        },
+        "courses/cms/course_detail.html course_prerequisites": {
+            "name": _("Prerequisites"),
+            "plugins": ["TextPlugin"],
+        },
+        "courses/cms/course_detail.html course_team": {"name": _("Course team")},
```

Comment:
We should enforce the addition of only PersonPlugin inside this placeholder

Reply:
You marked the wrong line, but yep it was planned, still we don't have PersonPlugin yet :)

Reply:
Actually we have! the PR was merged but an unfortunate "push -f" on master wiped it out. 
I rebased the commits and recreated the PR so we can merge it again: https://github.com/openfun/richie/pull/267